src/components/inputs/LInputBorder.vue

```diff
@@ -23,6 +23,9 @@ export default {
       isFocused: false
     }
   },
+  mounted () {
+    console.warn('LInputBorder is deprecated and will be removed in the future')
+  },
```

Comment:
pra não ficar repetindo a frase sempre, poderia usar um função pra ter o warning

Reply:
e talvez colocar quando se for o caso, por exemplo, próxima major release

Reply:
Tinha falado sobre isso com o MP, pode ser uma boa, mas não sei se precisa. Como a ideia é jogar fora os componentes que não fazem sentido e depois não precisar ficar depreciando, não vejo sentido em criar uma estrutura pra isso, seja ela uma função ou até um mixin